docs/examples/movie-reviews-graph-rag.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<p>Is there something more specific we think this example shows? Is it <em>just</em> a case study on using rotten tomatoes?</p><p> </p><p>Specifically, what do we think this example teaches? Does it show how to use existing structured information? etc.</p><p> </p><p>Regarding the strategy, I wonder if we want to make it clear that storing the reviews with the metadata is generally useful (eg., even if you were doing vector only) since it lets you filter, etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datastax/graph-rag/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>